<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>632</width>
    <height>485</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="title">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;YouTube Trending Prediction&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="info">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Scenario: &lt;br/&gt;You made a video and are certain that it will go trending.&lt;br/&gt;This application will help make it go trending within 24 hours.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="title_label">
      <property name="minimumSize">
       <size>
        <width>319</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>Title:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="title_input">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="descr_label">
      <property name="text">
       <string>Description</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="descr_box"/>
    </item>
    <item>
     <widget class="QLabel" name="tag_label">
      <property name="text">
       <string>Tags (separated with &quot;,&quot;)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="tags_input"/>
    </item>
    <item>
     <widget class="QLabel" name="cate_label">
      <property name="text">
       <string>Category</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="cate_combo"/>
    </item>
    <item>
     <widget class="QPushButton" name="run_button">
      <property name="text">
       <string>Run</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="quit_button">
      <property name="text">
       <string>Quit</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
